Azure/HelloCloud.Headless/README.md

```diff
@@ -0,0 +1,14 @@
+Windows 10 IoT Core sample code
+===============
+
+## HelloCloud.Headless
+
+This sample is a headless (no GUI) variant of the HelloCloud sample.
```

Comment:
Add something like you have in the GUI readme:  This sample demonstrates how to send a single message to Azure IoT Hub

Otherwise, someone would have to go to the GUI sample to know what the headless sample does.